File: app/src/main/java/com/example/a20200507_jessosborn_nycschools/Model/RetrofitEndpoint.java
package com.example.a20200507_jessosborn_nycschools.Model;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;

interface RetrofitEndpoint {

    //Base URL = https://data.cityofnewyork.us/resource/

    @GET("f9bf-2cp4.json")
    Call<SchoolsList> getSchoolsList();     //Might have to make this Call<List<Schools>

    //TODO: create callback for SAT data
}

File: app/src/main/java/com/example/a20200507_jessosborn_nycschools/Model/RetrofitEndpoint.kt
package com.example.a20200507_jessosborn_nycschools.Model

import retrofit2.Call
import retrofit2.http.GET

interface RetrofitEndpoint {

    //Base URL = https://data.cityofnewyork.us/resource/

    @get:GET("f9bf-2cp4.json")
    val schoolsList: Call<List<SchoolSATData>?>?
}